src/main.go

```diff
@@ -143,8 +116,14 @@ func main() {
 	go killListener(c, beforeExit)
 
 	var err error
-	if config, err = loadConfig(*configFilePath); err != nil {
-		logrus.WithError(err).Fatal("failed to load config file")
+
+	config, err = loadConfigFromEnv()
+	if err != nil {
+		logrus.WithError(err).Info("failed to load config from environment variables, trying to load from file")
+
+		if config, err = loadConfigFromPath(*configFilePath); err != nil {
+			logrus.WithError(err).Fatal("failed to load config file")
+		}
 	}
```

Comment:
Could this please be wrapped in a `loadConfig` function located in `config.go`?

Reply:
Sure! Done.